#goal definition
string  goalPerson
---
#result definition
string  faceRecognizedName
---
#feedback
string  faceDetectedName

